src/pages/notice/NoticePage.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  Container,
+  Title,
+  Text,
+  Group,
+  Rating,
+  Textarea,
+  Button,
+  Center,
+  Card,
+  Badge,
+} from '@mantine/core';
+import { Carousel } from '@mantine/carousel';
+import { useState } from 'react';
+
+const reviews = [
+  {
+    pseudo: 'John Doe',
+    date: '2023-02-01',
+    comment: 'La plateforme est très intuitive, je suis très satisfait !',
+    rating: 5,
+  },
+  {
+    pseudo: 'Jane Doe',
+    date: '2023-01-15',
+    comment:
+      'J’ai eu un problème avec ma réservation, mais le service client a été très réactif et m’a aidé à trouver une solution.',
+    rating: 4,
+  },
+  {
+    pseudo: 'Bob Smith',
+    date: '2022-12-20',
+    comment:
+      'Je suis un peu déçu par la qualité des services proposés, mais le rapport qualité-prix est correct.',
+    rating: 3,
+  },
+  {
+    pseudo: 'Alice Smith',
+    date: '2022-12-01',
+    comment:
+      'Je recommande vivement cette plateforme, j’ai passé un excellent séjour !',
+    rating: 5,
+  },
+  {
+    pseudo: 'Eva Brown',
+    date: '2022-11-15',
+    comment:
+      'Je suis très satisfaite de ma réservation, je n’hésiterai pas à revenir !',
+    rating: 5,
+  },
+];
+
+const NoticePage = () => {
+  const [comment, setComment] = useState('');
+  const [rating, setRating] = useState(0);
```

Comment:
add type for State like this : 
```suggestion
  const [comment, setComment] = useState<string>('');
  const [rating, setRating] = useState<number>(0);
```